test/acceptance/test/utils.go

```diff
@@ -339,12 +343,26 @@ func initAndCreateEmptyRepo(appRepoName string, IsPrivateRepo bool) string {
                             hub create %s %s`, repoAbsolutePath, repoAbsolutePath, GITHUB_ORG+"/"+appRepoName, privateRepo))
 	session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
 	Expect(err).ShouldNot(HaveOccurred())
-	Eventually(session).Should(gexec.Exit())
+	Eventually(session, 10).Should(gexec.Exit())
+	Expect(session.ExitCode()).Should(Equal(0))
+	Expect(session.Err).ShouldNot(gbytes.Say(fmt.Sprintf("mkdir: /tmp/%s: File exists", appRepoName)))
+	Expect(session.Err).ShouldNot(gbytes.Say("Existing repository detected"))
+	if os.Getenv("CI") == "true" {
+		Expect(session.Out).Should(gbytes.Say(fmt.Sprintf(`Initialized empty Git repository in /tmp/%s/.git/
+Updating origin
+https://github.com/%s/%s`, appRepoName, GITHUB_ORG, appRepoName)))
+	} else {
```

Comment:
Why do we have this if statement in the acceptance tests.  Shouldn't the result always be the same?